frontend/src/main/components/Commons/CommonsForm.js

```diff
@@ -70,7 +70,7 @@ function CommonsForm({initialCommons, submitAction, buttonLabel = "Create"}) {
     
     const testid = "CommonsForm";
     const curr = new Date();
-    const today = curr.toISOString().split('T')[0];
+    const today = curr.toLocaleDateString('en-CA');
```

Comment:
Oh I see.. It's the language. I'll change them to US english.

Reply:
I think its necessary to use canadian english because it formats the date as YYYY-MM-DD (and thats the format that the code is working with). American english uses the format MM/DD/YYYY. 

Would it be appropriate to keep it as canadian english?

Reply:
Ah, I see.    In that case, yes, but let's comment that so that we understand the rationale.  This will not necessarily be obvious to the reader.